src/Symfony/Component/Validator/Constraints/UlidValidator.php

```diff
@@ -81,6 +82,16 @@ public function validate(mixed $value, Constraint $constraint): void
                     ->setCode(Ulid::TOO_LARGE_ERROR)
                     ->addViolation();
             }
+        } elseif (Ulid::FORMAT_RFC_4122 === $constraint->format) {
+            if (!preg_match('/^.{8}-.{4}-.{4}-.{4}-.{12}$/', $value)) {
```

Comment:
we should use `[^.]` instead of just `.`

Reply:
Indeed, I added the related test case also